### PR TITLE
fixing module name, prompts and other bugs

### LINKED ISSUE #16: Terminal prompt is wrong
the lab guide says VS Code is installed on control node 1, but it's actually installed on the bastion

### PR DESCRIPTION
**Pull request by cbolz**
closes #16

### CODE REVIEW

File: content/ansible-collections/4-using-collections-from-tower/_index.md [hunk 1]
```diff
@@ -128,13 +128,13 @@ The inventory **Workshop Inventory** and the machine credentials **Workshop Cred
 
 ## Configure SCM Credentials
 
-Now we will configure the credentials to access our the Git repo on your control host via SSH. In the **RESOURCES** menu choose **Credentials**. Now:
+Now we will configure the credentials to access our the Git repo on your control host via SSH. In the **Resources** menu choose **Credentials**. Now:
```
Comment (goetzrieger) [inline content/ansible-collections/4-using-collections-from-tower/_index.md:132]: our the Git repo

File: content/ansible-collections/4-using-collections-from-tower/_index.md [hunk 2]
```diff
@@ -128,13 +128,13 @@ The inventory **Workshop Inventory** and the machine credentials **Workshop Cred
...
```
Comment (goetzrieger) [inline content/ansible-collections/4-using-collections-from-tower/_index.md:131]: our the Git repo

File: content/ansible-collections/4-using-collections-from-tower/_index.md [hunk 3]
```diff
@@ -155,33 +155,33 @@ MIIEpAIBAAKCAQEA2nnL3m5sKvoSy37OZ8DQCTjTIPVmCJt/M02KgDt53+baYAFu1TIkC3Yk+HK1
...
 You have now setup credentials to access the Git repo on your control host.
 
 ## Set up the Project
 
 It's time to set up the Tower Project pointing to your Git repository holding your Playbook and collections requirements file.
 
-- Go to **RESOURCES → Projects** in the side menu view click the ![plus](../../images/green_plus.png?classes=inline) button. Fill in the form:
-- **NAME:** Collections Repo
-- **ORGANIZATION:** Default
+- Go to **Resources → Projects** in the side menu view click the ![plus](../../images/green_plus.png?classes=inline) button. Fill in the form:
```
Comment (goetzrieger) [inline content/ansible-collections/4-using-collections-from-tower/_index.md:166]: Didn't the button change to blue now so we should change the image?

File: content/ansible-controller-advanced/3-awx-collection-intro/_index.md [hunk 4]
```diff
@@ -107,10 +107,10 @@ Run and test your playbook and verify everything works as expected, by logging i
 ## Create Machine Credentials
 
 {{% notice tip %}}
-SSH keys have already been created and distributed in your lab environment and `sudo` has been setup on the managed hosts to allow password-less login. When you SSH into a host as user **student{{< param "student" >}}** from **{{< param "internal_control" >}}** you will become user **ec2-user** on the host you logged in.
+SSH keys have already been created and distributed in your lab environment and `sudo` has been setup on the managed hosts to allow password-less login. When you become root on your bastion host by running `sudo -i` you can ssh into your managed nodes by using the pre-deployed SSH keys.
```
Comment (goetzrieger) [inline content/ansible-controller-advanced/3-awx-collection-intro/_index.md:110]: Keys are now distributed for direct SSH as lab-user

File: content/ansible-collections/4-using-collections-from-tower/_index.md (hunk 3 above)
Comment (cbolz) [reply content/ansible-collections/4-using-collections-from-tower/_index.md:166]: not when I was testing

**Comment by cbolz** [comment]
as discussed with Götz, this one gets closed

**Pull request closed without merging**